prow/cmd/crier/README.md

```diff
@@ -48,10 +48,45 @@ The actual report logic is in the [github report library](/prow/github/report) f
 
 You can enable the Slack reporter in crier by specifying the `--slack-workers=n` and `--slack-token-file=path-to-tokenfile` flags.
 
-The `--slack-token-file` flag takes a path to a file containing a Slack [*OAuth Access Token*](https://api.slack.com/docs/oauth). The access token can be obtained after installing 
-a [Slack app](https://api.slack.com/apps) for the *workspace*. The app should have the [`chat:write:bot`](https://api.slack.com/scopes/chat:write:bot) scope granted in order to [post messages](https://api.slack.com/methods/chat.postMessage) to the configured Slack *channel*.
+The `--slack-token-file` flag takes a path to a file containing a Slack [**OAuth Access Token**](https://api.slack.com/docs/oauth).
+
+The **OAuth Access Token** can be obtained as follows:
+1. Navigate to: https://api.slack.com/apps.
+1. Click **Create New App**; 
+1. Provide an **App Name** (e.g. Prow Slack Reporter) and **Development Slack Workspace** (e.g. Kubernetes).
+1. Click **Permissions**.
+1. Add the `chat:write:bot` scope using the **Scopes** dropdown and **Save Changes**.
+1. Click **Install App to Workspace**
+1. Click **Allow** to authorize the Oauth scopes.
+1. Copy the **OAuth Access Token** and save the token to a file.
+
+Once the *access token* is obtained and saved to a file, you can create a `secret` in the cluster using that file: 
+
+```shell
+kubectl create secret generic slack-token --from-file=token=/path/to/slack/token/file
+``` 
+
+Furthermore, to make this token available to **Crier**, mount the *slack-token* `secret` using a `volume` and set the `--slack-token-file` flag in the deployment spec.
+```yaml
+...
+spec:
+  containers:
+  - name: crier
+    image: gcr.io/k8s-prow/crier:v20191008-bbd30bc09
+    args:
+    - --slack-token-file=/etc/slack/token
```

Comment:
I bet someone will copy this, so maybe also add  `--dry-run=false` and `-slack-workers=1` here?

Not sure if we should mention this here, but if this is started the first time and not misconfigured, you will initially get num(ProwJobsInYourCluster) messages :grimacing:

Reply:
Modulo the filtering setup for the slack reporter, but may still be a high number

Reply:
👌 . Will update to a minimal configuration.

Reply:
> Not sure if we should mention this here, but if this is started the first time and not misconfigured, you will initially get num(ProwJobsInYourCluster) messages

Added a small note denoting this.